### PR TITLE
Remove use of vendor directory

### PR DESCRIPTION
**Pull request by gilbode**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -41,7 +41,7 @@ CRD_PATH=operator/config/crd/bases
 CODEGEN_PATH = k8s.io/code-generator
 
 .PHONY: build
-build: go-mod
+build:
```
Comment (markxnelson) [inline Makefile:44]: did you intentionally leave the `go-mod` target in the makefile?

Comment (gilbode) [reply Makefile:44]: Sort of - mainly b/c i didn't understand the implications of removing it given all the code generator related stuff

Comment (gilbode) [reply Makefile:44]: I was hoping some comment might explain why it could just be tossed :)

Comment (markxnelson) [reply Makefile:44]: there's one way to find out ;)

**Review by markxnelson — APPROVED** [review]

**Pull request merged**